### ISSUE TITLE
Hamster does not handle timezone transitions even when no task is running

### ISSUE THREAD

**Issue by moggers87** [projecthamster/hamster#346]
This is a problem I've seen a few times now:

1. have laptop powered on in UTC timezone
2. make sure all running timers are stopped
3. suspend laptop
4. move to BST timezone (either due to travel or due to daylight savings rules)
5. start a task in hamster and then stop it
6. start a new task and stop that too

After step 5, I see that the last task in hamster is 1 hour + whatever the actual time was. This continues to happen until I log out and in again - killing hamster-service and hamster-windows-service does not.

Version details:
Fedora 26
Gnome 3.24
hamster-time-tracker-2.0-0.10.rc1.fc26.noarch
Applet from [this PR](https://github.com/projecthamster/hamster-shell-extension/pull/174)

**Comment by ederag** [comment]
@moggers87 There have been some changes in this direction. 
Is it really fixed now ?

**Comment by moggers87** [comment]
I no longer use hamster.

**Comment by ederag** [comment]
Thanks, reopening then.
